spec/calculator.spec.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
This is our unit test file. It contains the tests for our code. It currently only contains 2 basic tests, but you should add some more.